### ISSUE TITLE
Shared folder not working after update to 4.3.3

### ISSUE THREAD

**Issue by jaraco** [utmapp/UTM#5506]
**Describe the issue**  

After upgrading from 4.2.5 to 4.3.3, the shared directory is no longer working.

I've powered off the machine and confirmed that the shared directory is pointing to my home directory and it's configured for Spice WebDAV.

I then boot up the Windows 11 ARM guest, but the shared drive remains the temporary directory:

```
 ~ @ dir z:
....
 Volume in drive Z has no label.
 Volume Serial Number is 0000-0000

 Directory of Z:\

2022-03-26  17:11    <DIR>          .
2022-03-26  17:11    <DIR>          ..
2022-03-26  17:11               320 README.txt
The parameter is incorrect.
鴐ጏ      <DIR>          .spice-clipboard
               1 File(s)            320 bytes
               3 Dir(s)   3,124,736,000 bytes free
 ~ @ cat z:\README.txt
You have not selected a shared directory. This is a temporary directory that can be deleted at any time. You can access these files on the host at ~/Documents/Public relative to UTM's sandbox (if enabled). To select a permanent shared directory, shut down the VM and select a shared directory from the VM details screen. ~ @
```

**Configuration**  
* UTM Version: 4.3.3
* macOS Version: 13.5
* Mac Chip (Intel, M1, ...): M1

**Debug log**  

[debug.log](https://github.com/utmapp/UTM/files/12207470/debug.log)

**Comment by jaraco** [comment]
I can see from the debug log that UTM is mounting the temporary directory and not the configured one.

**Comment by jaraco** [comment]
Downgrading to 4.2.5 restores the expected behavior.

**Comment by mx1up** [comment]
I also got a shared folder error after upgrading to 4.3.3: I simply could not start the VM anymore, QEMU claiming it did not have access to the folder. I checked the access rights but everything seemed fine. I cleared the shared folder configuration, and VM booted again afterwards.

**Comment by mifi** [comment]
FWIW I got this error when trying to share my `/Users/x/Public` folder. Sharing a different folder works.

**Comment by jaraco** [comment]
I confirmed this issue was fixed in 4.3.5, and based on the tag, the fix was first released in 4.3.4.
